frontend/src/components/Login.jsx

```diff
@@ -26,6 +26,7 @@ const Login = () => {
       })
       .catch((err) => console.log("Erorr in custom login", err));
   }
+  console.log(token)
```

Comment:
Typically you should remove all console.log before merging them into your main code.

Reply:
Kept there for ease in API testing. Will remove before moving to prod.